stl/inc/flat_set

```diff
@@ -661,6 +687,10 @@ private:
         return _My_pair._Get_first();
     }
 
+    _NODISCARD auto _Get_comp_v() const noexcept {
+        return _STD _Pass_fn(_My_pair._Get_first());
+    }
+

```

Comment:
the meaning of `_v` looks not very obvious (suitable for passing by value/"passed" by _Pass_fn); however I have difficulty finding a more suitable name here...

Reply:
How about `_Get_comp_for_passing`? Although it's a bit lengthy.

Reply:
Ooh, what about `_Pass_comp`?